libs/ledger-live-common/src/exchange/providers/swap.ts

```diff
@@ -203,14 +231,50 @@ function transformData(providersData) {
   return transformed;
 }
 
-export const getProvidersData = async () => {
-  const providersData = await network({
-    url:
-      "https://crypto-assets-service.api.ledger.com/v1/partners" +
-      "?output=name,signature,public_key,public_key_curve" +
-      "&service_name=swap",
+export const getProvidersData = async (): Promise<Record<string, ProviderData>> => {
+  const { data: providersData } = await network<ProvidersDataResponse>({
+    method: "GET",
+    url: "https://crypto-assets-service.api.ledger.com/v1/partners",
+    params: {
+      output: "name,signature,public_key,public_key_curve",
+      service_name: "swap",
+    },
   });
-  return transformData(providersData.data);
+
+  return transformData(providersData);
+};
+
+/**
+ * Retrieves the currency data for a given ID
+ * coins and tokens are litteraly the same on our case but we need to call the correct endpoint.
+ * @param currencyId The unique identifier for the currency.
+ * @param type Specifies the type of currency, either "tokens" or "coins".
+ * @returns A promise that resolves to the currency data including ID, serialized config, and signature.
+ */
+export const findExchangeCurrencyData = async (
```

Comment:
There is a `/currencies` that unifies both: https://crypto-assets-service.api.ledger.com/v1/currencies?id=bitcoin&output=id,exchange_app_config_serialized,exchange_app_signature